Tests/ConfidenceTests/Helpers/DebugLoggerTests.swift

```diff
@@ -0,0 +1,70 @@
+import Foundation
+
+@testable import Confidence
+
+internal class DebugLoggerTests: DebugLogger {
+    private let uploadSuccessCounter = ThreadSafeCounter()
+
+    func logEvent(action: String, event: ConfidenceEvent?) {
+        // no-op
+    }
+
+    func logMessage(message: String, isWarning: Bool) {
+        if message == "Event upload: HTTP status 200" {
```

Comment:
Is this enough to call it a success?
What if we send an event name that doesn't exist? Is the event going to be the same?

Reply:
Added a more granular check via string parsing, wdyt?